modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-topology-aware-lifecycle-operator-config_{context}"]
+= About the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of policies for one or more SNO clusters in a given time interval. The Operator allows you to control the update order of {rh-rhacm}-managed SNO clusters.
+
+The Operator supports the orchestration of the following update scenarios:
+
+* {product-title} y-stream and z-stream updates
+* Day two operations on y-stream and z-streams
+
+The `ClusterGroupUpgrade` CR defines the update that you want to apply to a group of clusters based on the following specifications:
+
+* Clusters in the group
+* Blocking `ClusterGroupUpgrade` CRs
+* Applicable list of managed policies
+* Number of concurrent updates
+* Applicable canary updates
+* Desired {product-title} version
+* Desired Operator version
```

Comment:
Just to see if my understanding is correct:
The versions are specified in the managed policies. These managed policies are created with the `PolicyGenTemplate`. After the policies are created, they can be listed in the `managedPolicies` of `ClusterGroupUpgrade` CR. `ClusterGroupUpgrade` is basically used for the automation of the update procedure (defines clusters, canaries, batches, timing, etc). 

So it would be more accurate to delete the versions in this bulleted list and say that the CR defines the remediation plan, instead of the update. Like this:

> The `ClusterGroupUpgrade` custom resource (CR) defines the **remediation plan** that you want to apply to a group of clusters based on the following specifications:
> * Clusters in the group
> * Blocking `ClusterGroupUpgrade` CRs
> * Applicable list of managed policies
> * Number of concurrent updates
> * Applicable canary updates
> * Pre- and post-update actions
> * Update timing